Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[workspace]
+
+members = [
+    "multisig-wallet",
+]
+
```

Comment:
I don't think we really want this to be a Rust workspace, since it's not actually a bunch of Rust projects. It's a bunch of Sway projects.